src/inc/corcompile.h

```diff
@@ -750,13 +750,15 @@ enum CORCOMPILE_FIXUP_BLOB_KIND
 
 enum EncodeMethodSigFlags
 {
-    ENCODE_METHOD_SIG_UnboxingStub              = 0x01,
-    ENCODE_METHOD_SIG_InstantiatingStub         = 0x02,
-    ENCODE_METHOD_SIG_MethodInstantiation       = 0x04,
-    ENCODE_METHOD_SIG_SlotInsteadOfToken        = 0x08,
-    ENCODE_METHOD_SIG_MemberRefToken            = 0x10,
-    ENCODE_METHOD_SIG_Constrained               = 0x20,
-    ENCODE_METHOD_SIG_OwnerType                 = 0x40,
+    ENCODE_METHOD_SIG_UnboxingStub                              = 0x01,
+    ENCODE_METHOD_SIG_InstantiatingStub                         = 0x02,
+    ENCODE_METHOD_SIG_MethodInstantiation                       = 0x04,
+    ENCODE_METHOD_SIG_SlotInsteadOfToken                        = 0x08,
+    ENCODE_METHOD_SIG_MemberRefToken                            = 0x10,
+    ENCODE_METHOD_SIG_Constrained                               = 0x20,
+    ENCODE_METHOD_SIG_OwnerType                                 = 0x40,
+    ENCODE_METHOD_SIG_OwnerTypeIsTypeSpecWithVars               = 0x80,
+    ENCODE_METHOD_SIG_MethodInstantiationIsMethodSpecWithVars   = 0x100,
```

Comment:
Why can't we just use `ENCODE_METHOD_SIG_MethodInstantiation` and `ENCODE_METHOD_SIG_OwnerType`?